### DISCUSSION TITLE
Cannot start Milvus on minikube

### DISCUSSION THREAD

**Discussion by sutcalag** [milvus-io/milvus#15277] [Q&A and General discussion]
My test env is a mbp with intel chip.

I follow guide from https://milvus.io/docs/v2.0.0/install_standalone-helm.md to install milvus but the pod keeps on not ready
![image](https://user-images.githubusercontent.com/83750738/149899636-c3edb895-b3c9-42d1-aef8-de23f7fc31a8.png)

Attachment is my locall log
[kubectl.log](https://github.com/milvus-io/milvus/files/7887406/kubectl.log)
[minikube.log](https://github.com/milvus-io/milvus/files/7887408/minikube.log)

**Comment by LoveEachDay** [comment]
@sutcalag minio pod is`pending`. Please make sure `minio` and `etcd` pods are running. 
Use the following command to check why minio pod is pending:
`kubectl describe pods my-release-minio-54fc79dbdf-ckcbn`
